lib/openactive/validators/date_interval_validator.rb

```diff
@@ -0,0 +1,44 @@
+module OpenActive
+  module Validators
+    class DateIntervalValidator < BaseValidator
+      # Coerce given value to the type the validator is validating against.
+      # PLEASE NOTE: no checks are performed on the given $value.
+      # It is therefore recommended to call the "run" method first before this.
+      #
+      # @param value mixed The value to coerce.
+      # @return mixed The same value.
+      # @throws ::Exception When $value cannot be parsed as a ::DateInterval
+      def coerce(value)
+        # instanceof does not throw any error
+        # if the variable being tested is not an object,
+        # it simply returns FALSE.
```

Comment:
Not sure this comment is relevant any more

Reply:
Removed all these.